### PR TITLE
Add media queries for phones and tablet screens

### LINKED ISSUE #13: Responsiveness for phones and tablet screens
Media queries for smaller screen sizes

### PR DESCRIPTION
**Pull request by Dustin-Mikusko**
I've added media queries for smaller screens. I have the columns to go to 2 at 900px (seen on line 187 on the styles.css) and 1 column at 650px (line 195). This PR will close #13

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -181,4 +180,22 @@ h4 {
 p {
   font-weight: 200;
   font-size: 1em;
-}
```
Comment (adamsjr8576) [inline src/css/styles.css]: Does this mean that the closing tags of the p element were deleted?

File: src/css/styles.css [hunk 2]
```diff
@@ -181,4 +180,22 @@ h4 {
...
+  }
+}
+
+@media only screen and (max-width: 650px) {
+  .grid {
+    column-count: 1;
+    column-gap: 1em;
+    column-fill: balance;
+  }
+}
```
Comment (adamsjr8576) [inline src/css/styles.css]: Nice work! This looks awesome and I think that the shifting from 3 to 2 to 1 columns is definitely the best way to go about this!

File: src/css/styles.css (hunk 1 above)
Comment (Dustin-Mikusko) [reply src/css/styles.css]: There was a blank line there that I deleted to bring the closing tag up.

**Comment by adamsjr8576** [comment]
Looks like once we get packery working there won't be a need for media queries to break up the columns so we will close this one.

**Pull request closed without merging**